src/icon-manifest/manifestMerger.ts

```diff
@@ -1,49 +1,39 @@
 import * as _ from 'lodash';
-import {
-  IFileCollection,
-  IFolderCollection,
-  IExtensionCollection,
-  IExtension,
-  FileFormat,
-  IIconSchema,
-  IIconGenerator,
-  IFileDefault,
-  IFolderDefault,
-} from '../models';
+import * as model from '../models';
```

Comment:
I generally try to not use generic imports. Basically because I'm used to think about `tree-shaking` (which is not our case). Personally, I don't like much the verbose style. Maybe I'm used to .NET, where you usually import namespaces in order to not do what you're proposing here. 

Anyway, as this is a matter of style preference I'm not going to discuss it. 😉